olp-cpp-sdk-core/src/cache/DefaultCacheImpl.cpp

```diff
@@ -526,7 +526,7 @@ void DefaultCacheImpl::InitializeLru() {
   }
 
   OLP_SDK_LOG_INFO_F(kLogTag,
-                     "LRU cache initialized, items=%zu, time=%" PRId64 " ms",
+                     "LRU cache initialized, items=%zu, time=%" PRId64 " us",
```

Comment:
```suggestion
                     "LRU cache initialized, items=%zu, time=%" PRId64 "us",
```

Same as below on line 635